shaded/pom.xml

```diff
@@ -25,6 +25,7 @@
   <modules>
     <module>client</module>
     <module>hadoop</module>
+    <module>ozone</module>
```

Comment:
is shading this module required?
In UFS, by default all dependencies are loaded by a separate class loader, do we still see the exception you described?

```
Add a new ozone shade module to avoid classCaseException for the following code from org.apache.hadoop.ipc.ProtobufRpcEngine#invoke
      final Message theRequest = (Message) args[1];
```

Reply:
@apc999 I have to shade ozone, because ozone ufs module dependent hdfs ufs module, so the `Message` in the `ProtobufRpcEngine` has been shaded by hdfs shade module.